client/src/Pages/Charts/Charts.jsx

```diff
@@ -11,14 +12,14 @@ function Charts() {
 
   const grabProfile = async () => {
```

Comment:
please move grabProfile() logic to Profile.jsx

Reply:
done